index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awsome books</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <h1>Awesome books</h1>
+  </header>  
+  <main>
+    <section class="books">
+      <!-- <div class="book">
+        <p class="title">Lorem ipsum</p>
+        <p class="author">Tester Testy</p>
+        <button class="remove">Remove</button>
+        <hr>
+      </div> -->
```

Comment:
- [ ] Please kindly remove this commented lines of code to make your codebase cleaner according to best practises. Thanks. 👍